### ISSUE TITLE
Launching `localhost` in the browser only works on Windows.

### ISSUE THREAD

**Issue by sabbaghpierre** [Hawmex/dawn#10]
Hi, I'm getting this error whenever I create a new project and try running it.

```
+ Copied assets and index.html to .dawn/dev...
+ Compiled main.dart.

Starting server...
Server running on http://localhost:5500.
Unhandled exception:


#0      runProcess (package:dawn/src/foundation/process_runner.dart:16:7)
#1      CompileCommand._runServer (package:dawn/src/commands/compile_command.dart:77:5)
<asynchronous suspension>
#2      CompileCommand._runCommand (package:dawn/src/commands/compile_command.dart:58:21)
<asynchronous suspension>
#3      CompileCommand.run (package:dawn/src/commands/compile_command.dart:42:5)
<asynchronous suspension>
#4      CommandRunner.runCommand (package:args/command_runner.dart:209:13)
<asynchronous suspension>
```

**Comment by Hawmex** [comment]
Could you please provide your OS information? (MacOS/Windows/Linux/etc.)

**Comment by sabbaghpierre** [comment]
I'm running Arch Linux x86_64, with kernel 5.19.3-arch1-1

**Comment by Hawmex** [comment]
Found the problem. Will be fixed in the next release.

**Comment by Hawmex** [comment]
> I'm running Arch Linux x86_64, with kernel 5.19.3-arch1-1

It should be fixed in `5.3.1`. I'll keep this open until you provide feedback.
